tox.ini

```diff
@@ -69,6 +70,25 @@ commands =
 
     pytest python
 
+[testenv:examples]
+# this environement runs the examples for Python and Pytorch bindings
+deps =
+    wheel
+    cmake
+
+    numpy
+    torch
+    pytest
+
+passenv=
+    PIP_EXTRA_INDEX_URL
```

Comment:
Is this explicitly required? I did not set it for other projects, and `PIP_EXTRA_INDEX_URL` seems to be passed through without any issue

Reply:
IDK, it was not working but perhaps I made a mistake. Doesn't hurt, that for sure...